src/infrastructure/utils/password.py

```
@@ -0,0 +1,17 @@
+import jwt
```

Ruff (0.8.2)

`jwt` imported but unused

Remove unused import: `jwt`

(F401)